src-vue/src/App.vue

```diff
@@ -34,9 +36,6 @@ export default {
     this.$root!.$i18n.locale = lang;
   },
   methods: {
-    async toggleMaximize() {
-      await appWindow.toggleMaximize();
-    },
```

Comment:
Is the reason for removing maximise to make space for the notification icon? ^^

Reply:
Yep, and also because I don't like the icon + window can still be maximized by double-clicking the menu bar ;)

Reply:
Aight, in that case I'll just make a separate PR with that change rq to get the diff down a bit more ^^

Reply:
#617